### PR TITLE
Update PowerShell Module Dependency Docs

### LINKED ISSUE #2795: The type initializer for 'Microsoft.WinGet.Client.Common.BaseClientCommand' threw an exception.
### Brief description of your issue

Installed the preview release that supports the PowerShell module. Commands like `Get-WingetSource` fail with error `The type initializer for 'Microsoft.WinGet.Client.Common.BaseClientCommand' threw an exception.`. Looking at the complete exception I wonder if there are leftover hard-coded references.

image

### Steps to reproduce

Installed the 1.4.351 preview release and the PowerShell module. Import the module, try to run a command like `Get-Wingetsource`.

### Expected behavior

Commands should work without error.

### Actual behavior

image

### Environment

```shell
Windows Package Manager v1.4.3531
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19044.2130
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.19.3531.0

Logs: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir

User Settings: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\settings.json

Links
---------------------------------------------------------------------------
Privacy Statement   https://aka.ms/winget-privacy
License Agreement   https://aka.ms/winget-license
Third Party Notices https://aka.ms/winget-3rdPartyNotice
Homepage            https://aka.ms/winget
Windows Store Terms https://www.microsoft.com/en-us/storedocs/terms-of-sale

PowerShell module 0.0.1 alpha
```

### PR DESCRIPTION
**Pull request by ryfu-msft**
Resolves #2795 

Changes:
Updates the PowerShell module documentation to explicitly identify the required dependency on the Microsoft Visual C++ Redistributable package in order to run the cmdlets

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-cli/pull/2797)

### CODE REVIEW

File: src/PowerShell/Microsoft.WinGet.Client/README.md [hunk 1]
```diff
@@ -6,6 +6,15 @@ This project contains the source code for building the Windows Package Manager P
 
 After building the project solution, the `Microsoft.WinGet.Client` PowerShell module can be found in the output directory in the `PowerShell` folder. For example if you built the project as x64 release, you should expect to find the module files in `$(SolutionDirectory)/src/x64/Release/PowerShell`.
 
+## Prerequisites
+
+The PowerShell module has a dependency on the Microsoft Visual C++ Redistributable package which can be downloaded [here](https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist?view=msvc-170) or installed through WinGet by running the following command:
```
Comment (florelis) [inline src/PowerShell/Microsoft.WinGet.Client/README.md:11]: Don't use "here" for links. Use a more descriptive text. Think about how the link looks for a screen reader listing the links in a page. I don't know what to use, though... Maybe "from [Microsoft Learn]"?

Also, don't include `/en-US/` in the URI. Let the browser pick the appropriate locale.

Comment (florelis) [inline src/PowerShell/Microsoft.WinGet.Client/README.md:11]: > WinGet

Are we no longer calling it "Windows Package Manager" in docs?

**Review by florelis — APPROVED** [review]

File: src/PowerShell/Microsoft.WinGet.Client/README.md (hunk 1 above)
Comment (denelon) [inline src/PowerShell/Microsoft.WinGet.Client/README.md:11]:
```suggestion
The PowerShell module has a dependency on the [Microsoft Visual C++ Redistributable](https://learn.microsoft.com/cpp/windows/latest-supported-vc-redist?view=msvc-170) package which can be downloaded or installed through WinGet by running the following command:

```

Comment (florelis) [inline src/PowerShell/Microsoft.WinGet.Client/README.md:11]: What version do I need to download?

Comment (robinmalik) [reply src/PowerShell/Microsoft.WinGet.Client/README.md]: @florelis I've got the [Microsoft Visual C++ 2015-2022 Redistributable, v14.34.31931](https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist?view=msvc-170) on my W11 machine and it works for me (well, it imports and Get-WinGetPackage runs).

Comment (ryfu-msft) [reply src/PowerShell/Microsoft.WinGet.Client/README.md]: I added the word "latest" and the id in the winget example points to 2015.

Comment (ryfu-msft) [reply src/PowerShell/Microsoft.WinGet.Client/README.md]: Removed locale from URI

Comment (florelis) [reply src/PowerShell/Microsoft.WinGet.Client/README.md]: Thanks @robinmalik. I actually meant it as a rhetorical question for @ryfu-msft to specify it in the doc, which he did

Comment (ryfu-msft) [reply src/PowerShell/Microsoft.WinGet.Client/README.md]: Changed to suggested for naming consistency.

**Pull request merged**